client/app/components/packages/landuse-form/proposed-actions.js

```diff
@@ -0,0 +1,12 @@
+import Component from '@glimmer/component';
+
+export default class PackagesLanduseFormProposedActionsComponent extends Component {
+  requiredActionCodes = ['ZS', 'ZA', 'ZC', 'CM', 'LD', 'RA', 'RC', 'RS', 'SD', 'SA', 'SC'];
+
+  get projectHasRequiredActions() {
+    const projectActions = this.args.form.data.landuseActions;
```

Comment:
Stray observation: I wish we had a way to document the required arguments for a component. For example, this would fail fantastically without a form argument.

Also, this component is designed to know quite a bit about the shape of the form object itself — it needs to know about `.data` and expects the block param to have all sorts of things. Not necessarily a bad thing, but I wonder how we can make this clearer — perhaps there's a Component baseclass we could use to signify this is a FormSectionComponent.

Reply:
That's a good point, and it seems like a lot of new components have display requirements similar to this so maybe we can chat at engineering standup about best ways to approach/standardize this going forward. Maybe something similar to how we're doing optionsets? Where we store the requirements in other files (wondering too if some components might have similar requirements)